README.md

```diff
@@ -20,7 +20,7 @@ It can be found [here](https://github.com/SubstraFoundation/substra)
 
 You will need to install it thanks to the `pip` binary.
 
-# Run the tests
+# Run the tests on the remote backend
```

Comment:
The "remote backend" name is misleading because it implies that the backend runs on a "remote" machine, which is often not the case, especially on local environments.

Can we consider a different naming scheme?

e.g.

- deployed backend / mock backend
- backend / mock backend
- something else?

Reply:
I agree, "mock" backend is better than "local". Since it is also "local" in Substra, let's change all of it at once when the features are merged.